smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/commit/KafkaThrottledLatestProcessedCommit.java

```diff
@@ -47,6 +47,7 @@
     private final int unprocessedRecordMaxAge;
     private final int autoCommitInterval;
     private volatile long timerId = -1;
+    private final Collection<TopicPartition> assignments = new CopyOnWriteArrayList<>();
```

Comment:
Just today morning, I realized I saw `CopyOnWriteArrayList` here. I think `Collections.newSetFromMap(new ConcurrentHashMap<>())` would probably be more appropriate, given that you only use `addAll`, `removeAll` and `contains`.

Reply:
Hum, very good point! 
Fancy a PR (this one was already merged).

Reply:
Sure, gimme a sec :-)

Reply:
#1261#1261